src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -175,10 +176,25 @@ public void Version_MatchesFixedVersion()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // Throws InvalidOperationException in Uap as NtQuerySystemInformation Pinvoke is not available
         public void WorkingSet_Valid()
         {
             Assert.True(Environment.WorkingSet > 0, "Expected positive WorkingSet value");
         }
+        
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.Uap)] 
+        public void WorkingSet_Valid_Uap()
+        {
+            try
+            {
+                var workingSet = Environment.WorkingSet;
+            }
+            catch(TargetInvocationException ex)
```

Comment:
This isn't right, we shouldn't be throwing TIE to the user. It's an artefact of our implementation (that Environment.WorkingSet is using reflection because of our layering). Environment.WorkingSet should catch TIE and unwrap it.